symplyphysics/laws/thermodynamics/heat_transfer_equation.py

```diff
@@ -0,0 +1,48 @@
+from sympy import (Eq, solve)
+from symplyphysics import (units, Quantity, Symbol, print_expression, validate_input,
+    validate_output)
+
+# Description
+# heat flow calculation Ф = λ * (T1-T2)/ d * S
+# where:
+# λ = material_thermal_conductivity (Watts/(K*m))
+# T1 = inside_temperature (C')
+# T2 = outside_temperature (C')
+# d = wall_thickness (m)
+# S = wall_area (m2)
```

Comment:
Is this law limited to only walls? What if the structure is spherical, not rectangular?